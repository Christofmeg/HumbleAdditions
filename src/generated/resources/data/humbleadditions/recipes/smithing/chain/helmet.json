{
  "type": "minecraft:smithing",
  "addition": {
    "item": "minecraft:iron_block"
  },
  "base": {
    "item": "minecraft:leather_helmet"
  },
  "result": {
    "item": "minecraft:chainmail_helmet"
  }
}