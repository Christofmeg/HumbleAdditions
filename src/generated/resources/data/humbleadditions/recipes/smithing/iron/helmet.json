{
  "type": "minecraft:smithing",
  "addition": {
    "item": "minecraft:iron_ingot"
  },
  "base": {
    "item": "minecraft:golden_helmet"
  },
  "result": {
    "item": "minecraft:iron_helmet"
  }
}