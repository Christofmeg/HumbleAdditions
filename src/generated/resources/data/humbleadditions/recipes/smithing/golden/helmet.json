{
  "type": "minecraft:smithing",
  "addition": {
    "item": "minecraft:gold_block"
  },
  "base": {
    "item": "minecraft:chainmail_helmet"
  },
  "result": {
    "item": "minecraft:golden_helmet"
  }
}